pmd-java/src/test/java/net/sourceforge/pmd/lang/java/symboltable/BaseParserTest.java

```diff
@@ -15,20 +12,18 @@
  * Base class for tests that usually need processing stages to run when
  * parsing code.
  */
-public abstract class BaseNonParserTest {
+public abstract class BaseParserTest {
```

Comment:
Maybe this test base class should be moved a package up.... so that it is `net.sourceforge.pmd.lang.java.BaseParserTest`. It's used in different subpackages (e.g. `n.sf.pmd.lang.java.ast`) and not only in symboltable.